openshift_scalability/ci/scripts/create-loaded-projects.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash 
+
+if [ "$#" -ne 1 ]; then
+  echo "syntax: $0 <TYPE>"
+  echo "<TYPE> should be either golang or python"
+  exit 1
+fi
+
+TYPE=$1
+
+function python_clusterloader() {
+  MY_CONFIG=../../config/configCreateLoadedProjects.yaml
+  python_v=$(python --version 2>&1 | sed 's/.* \([0-9]\).\([0-9]\).*/\1\2/')
+  if [[ $python_v -lt "29" ]]; then
+    echo "Running python 2";
+  else
+    echo "Have python version: $(python --version 2>&1); Need to run with python 2: exiting"
+    exit
+  fi
+
+  python ../../cluster-loader.py -f $MY_CONFIG
+}
+
+function golang_clusterloader() {
+
+  # start GoLang cluster-loader
+  export KUBECONFIG=${KUBECONFIG-$HOME/.kube/config}
+  cur_loc=$(pwd)
+
+  MY_CONFIG=$cur_loc/../../config/golang/configCreateLoadedProjects.yaml
+
+  VIPERCONFIG=$MY_CONFIG openshift-tests run-test "[sig-scalability][Feature:Performance] Load cluster should populate the cluster [Slow][Serial]"
```

Comment:
Given the relative paths inside of config/golang/configCreateLoadedProjects.yaml this failed for me when I ran it from ci/scripts directory.   Which has been a weak point of golang CL and our directory structure.   I had to add the following line above the call to openshift-tests:
  cd ../../content
to get the ./<template-name> templates in configCreateLoadedProjects.yaml to work. 

We can either update the script or the yaml file, but I think we should be able to run the script from ci/scripts.